themes/jb/layouts/partials/meta.html

```diff
@@ -1,9 +1,9 @@
 <!-- Meta Ttitles -->
-<title itemprop="name">{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}</title>
-<meta property="og:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
-<meta name="twitter:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
-<meta itemprop="name" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
-<meta name="application-name" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
+<title itemprop="name">{{ .Title -}}{{- if .Title }} | {{end}}{{ if ne .Title (print .CurrentSection.Title ` ` .Params.episode) }}{{ .Params.show_name -}}{{- if .Params.show_name }} {{ if .Params.episode }}{{ .Params.episode -}}{{end}} | {{end}}{{end }}{{ .Site.Title }}</title>
+<meta property="og:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ if ne .Title (print .CurrentSection.Title ` ` .Params.episode) }}{{ .Params.show_name -}}{{- if .Params.show_name }} {{ if .Params.episode }}{{ .Params.episode -}}{{end}} | {{end}}{{end}}{{ .Site.Title }}" />
+<meta name="twitter:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ if ne .Title (print .CurrentSection.Title ` ` .Params.episode) }}{{ .Params.show_name -}}{{- if .Params.show_name }} {{ if .Params.episode }}{{ .Params.episode -}}{{end}} | {{end}}{{end}}{{ .Site.Title }}" />
+<meta itemprop="name" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ if ne .Title (print .CurrentSection.Title ` ` .Params.episode) }}{{ .Params.show_name -}}{{- if .Params.show_name }} {{ if .Params.episode }}{{ .Params.episode -}}{{end}} | {{end}}{{end}}{{ .Site.Title }}" />
+<meta name="application-name" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ if ne .Title (print .CurrentSection.Title ` ` .Params.episode) }}{{ .Params.show_name -}}{{- if .Params.show_name }} {{ if .Params.episode }}{{ .Params.episode -}}{{end}} | {{end}}{{end}}{{ .Site.Title }}" />
```

Comment:
Sure thing. The reason I avoid `with` here was simply because the scoping change caused it read a bit strange to me. That is, `{{ with .Title }}` assigns `.Title` to `.`, and so it read strange to me that `{{ . }}` only contains the contents of `.Title` (here, a string), where everywhere else, I expect `{{ . }}` to mean some _object_.  That said, that's just me - I can make this change. No problem at all.

I'll adjust/accept this and repush/squash.

Reply:
Actually, I'm not sure this still applies with the revised logic that I used in order to make it a bit more organized/clear in response to the previous comment from @CGBassPlayer. PTAL and let me know if using `with` here still makes sense.